gordon_janitor/main.py

```diff
@@ -167,9 +167,9 @@ def run(config_root):
     plugin_names, plugins, errors, plugin_kwargs = plugins_loader.load_plugins(
         config, plugin_kwargs)
 
-    if errors:
-        base_msg = 'Plugin was not loaded:'
-        _log_or_exit_on_exceptions(base_msg, errors, debug_mode)
+    for err_plugin, exc in errors:
```

Comment:
Are we testing these lines directly?

Reply:
Can you clarify?  I'm not sure what you mean.

Reply:
Are we testing what happens when a plugin doesn't load correctly?  The changes you made to the tests are all downstream of this.

Reply:
I'm still don't understand what you mean.  Can you point to the code that you think needs coverage?

Reply:
So I'm thinking of tests like this https://github.com/spotify/gordon-janitor/blob/develop/tests/unit/test_main.py#L161

Should we test more explicitly for the outcome of these lines?

Reply:
The two tests I have updated in the PR test the output of the log message from this function. I do not see why we would need to test it somewhere else as well.